assets/js/services/Nav.js

```diff
@@ -2,13 +2,25 @@ const $ = require('jquery');
 
 class Nav {
     constructor() {
-        $(function() {
-            $('.nav-item a').click(function() {
```

Comment:
I think this styles the tabs for activate of the raw block and raw transactions as opposed the the main nav.
Probably doesn't belong in the Nav.js but it used the .nav selector so was placed here.